src/components/Home/components/SinglePost/index.js

```diff
@@ -60,14 +62,24 @@ const Index = ({
           ) : (
             <AiOutlineHeart onClick={() => setIsLiked(!isLiked)} />
           )}
-          {likes >= 1000 ? `${likes / 1000}k` : likes}
+          {isLiked
+            ? likes >= 10000
+              ? `${((likes + 1) / 1000).toString().substring(0, 4)}k`
+              : likes + 1
+            : likes >= 10000
+            ? `${(likes / 1000).toString().substring(0, 4)}k`
+            : likes}
```

Comment:
having more than one ternary conditions is not recommended, it is better to create a function here, like `getLikesCount` at the component body and return correct value there. that function will be something like this:


```js
const getLikesCoutn = () => {
   const makeShortedNumber = (number) => return  number >= 10000
              ? `${((number) / 1000).toString().substring(0, 4)}k`
              : number;

   if(isLiked) {
      return makeShortedNumber(likes + 1);
   } else {
      return makeShortedNumber(likes);
   }
}
```
This one is really simpler